app/controllers/series_controller.rb

```diff
@@ -108,13 +108,9 @@ def edit
   # POST /series.json
   def create
     @series = Series.new(permitted_attributes(Series))
-    if @series.course.present?
-      authorize @series.course, :add_series?
-    else
-      @series.errors.add(:course_id, I18n.t('errors.messages.blank'))
-    end
+    authorize @series, :create?
```

Comment:
```suggestion
    authorize @series
```

If you don't pass the second parameter, it is inferred from the action's name (in this case we are in the `create` action of the controller, so it should infer `create?`.